packages/vow/vat.js

```diff
@@ -28,13 +30,17 @@ export const defaultPowers = harden({
 /**
  * Produce SwingSet-compatible vowTools, with an arbitrary Zone type
  *
- * @type {typeof rawPrepareVowTools}
+ * @type {typeof prepareBasicVowTools}
  */
 export const prepareSwingsetVowTools = (zone, powers = {}) =>
-  rawPrepareVowTools(zone, { ...defaultPowers, ...powers });
+  prepareBasicVowTools(zone, { ...defaultPowers, ...powers });
+harden(prepareSwingsetVowTools);
 
-/** @deprecated */
-export const prepareVowTools = prepareSwingsetVowTools;
+/**
+ * Reexport as prepareVowTools, since that's the thing that people find easiest
+ * to reach.
+ */
+export { prepareSwingsetVowTools as prepareVowTools };
```

Comment:
not a blocker but consider leaving `prepareVowTools` deprecated. It lacks any indication of which kind it is.

Reply:
Instead of deprecating `prepareVowTools`, I updated the `README.md` to use it exclusively.  It's the one that "just works" on Agoric.  Given an `@agoric` scope, I think a reasonable expectation is that the packages within it will work well together without having to write `Agoric` or `SwingSet` over and over.

If people want to use it with a locked-down but SwingSet-incompatible platform (Metamask ocap-kernel, I'm looking at you), `prepareVatTools` can be wrapped to default the `powers` argument to use that platform's features.